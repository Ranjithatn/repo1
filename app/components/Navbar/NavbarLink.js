import React from "react";

const NavbarLink = ({ className, id, children }) => {
  return (
    <div
      className={`${"navbar-link" + (className ? `${" " + className}` : "")}`}
    >
      {children}
    </div>
  );
};

export default NavbarLink;
