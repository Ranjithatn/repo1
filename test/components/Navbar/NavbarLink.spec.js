import React from "react";
import toJson from "enzyme-to-json";
import NavbarLink from "../../../app/components/Navbar/NavbarLink";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<NavbarLink />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <NavbarLink />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
  
});
