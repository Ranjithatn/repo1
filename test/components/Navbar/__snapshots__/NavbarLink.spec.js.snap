// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarLink /> should render without crashing 1`] = `
<NavbarLink>
  <div
    className="navbar-link"
  />
</NavbarLink>
`;
